backend/sftp/sftp.go

```diff
@@ -1171,6 +1171,11 @@ func (f *Fs) mkdir(ctx context.Context, dirPath string) error {
 	err = c.sftpClient.Mkdir(dirPath)
 	f.putSftpConnection(&c, err)
 	if err != nil {
+		ok, _ := f.dirExists(ctx, dirPath)
+		if ok {
+			fs.LogPrintf(fs.LogLevelWarning, f, "directory %q exists after Mkdir is attempted", dirPath)
```

Comment:
Can you use one of the established log levels here fs.Debugf/Infof/Logf/Errorf - we only document use of those log levels.

I think `fs.Debugf` is fine here - we've created the directory and it exists.